.dockerignore

```diff
@@ -1,4 +1,3 @@
-.git*
```

Comment:
Git tag and Git commit is set during build: https://github.com/eset/grpc-rest-proxy/blob/main/Makefile#L2